content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -96,7 +96,7 @@ So, the example NetworkPolicy:
    * any pod in the "default" namespace with the label "role=frontend"
    * any pod in a namespace with the label "project=myproject"
    * IP addresses in the ranges 172.17.0.0–172.17.0.255 and 172.17.2.0–172.17.255.255 (ie, all of 172.17.0.0/16 except 172.17.1.0/24)
-3. allows connections from any pod in the "default" namespace with the label "role=db" to CIDR 10.0.0.0/24 on TCP port 5978
+3. allows connections to any pod in the "default" namespace with the label "role=db" to CIDR 10.0.0.0/24 on TCP port 5978
```

Comment:
No, the text is correct as written; the example NetworkPolicy sets up both ingress and egress rules for the `role=db` pods, and the last item is explaining an egress rule.

Perhaps some additionally wording or emphasis could be added to clarify that.

Reply:
@danwinship Thank you for the review :) 

I will rephrase it to clear the confusion may be. Will update the PR soon 👍

Reply:
@danwinship I rephrased the sentence to clear the confusion. Can you review, please.

@cody-clark